__tests__/app.test.js

```diff
@@ -163,7 +163,7 @@ describe("app", () => {
     });
     test("201: POST - adds an object onto comments with the properties username and body and responds with the posted comment", () => {
       const newComment = {
-        username: "HappyJuice123",
+        username: "mallionaire",
```

Comment:
Nice work on fixing these up, another test for posting a comment with a nonexistent user would be good.